django_app/redbox_app/redbox_core/types.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+from uuid import UUID
+
+from redbox_app.redbox_core import models
+
+
+class ChunkStatus:
```

Comment:
feels like DRF?

Reply:
I'm 50/50 on adding the extra weight that adding DRF will do, I'm hoping that this will be the only non-view url in the django app

Reply:
fair point

Reply:
Fully agree though that if we even think we'll add more in the future then we should add DRF, I'll raise it at tomorrows stand-up